conda/environments/cudf_dev_cuda9.2.yml

```diff
@@ -12,9 +12,9 @@ dependencies:
   - cmake_setuptools>=0.1.3
   - python>=3.6,<3.8
   - numba>=0.45.1
-  - pandas>=0.24.2,<0.25
+  - pandas>=0.25
```

Comment:
```suggestion
  - pandas>=0.25,<0.26
```

Reply:
Humm. I must have borked that in a merge. This could be the source of some of my last set of test failures.